### ISSUE TITLE
fedy-5.0.0-1.fc31: ships with pngs not resolved (content is git-lfs ref)

### ISSUE THREAD

**Issue by thestonewell** [rpmfusion-infra/fedy#8]
**Describe the bug**
After installing fedy-5.0.0-1.fc31 some of the icons are rendered incorrectly. Examples include but are not limited to the Firefox Developer Edition, etc etc.

**To Reproduce**
1. dnf install fedy
2. launch
3. observe that some icons show just as bang

**Expected behavior**
icons should show.

**Screenshots**
<img width="446" alt="fedy_20191110_160501" src="https://user-images.githubusercontent.com/5617372/68546152-ef89f500-03d3-11ea-9509-8f26336fe099.png">

**Additional context**
I tracked it down to the use of git-lfs. Seems during the build of the RPM we don't do a `git lfs fetch` and `git lfs checkout` to have the physical copies in the build tree.

Example after installing the RPM as is today:
```
cat /usr/share/fedy/plugins/firefox-developer.plugin/firefox-developer.png 
version https://git-lfs.github.com/spec/v1
oid sha256:66785fda053547f1411673acaa61c089b2f96c80d5c214d381ba8ba119233a11
size 3601
```

**Comment by kwizart** [comment]
Thx,  I think it would be good to remove the lfs dependency and convert it to a pure commit.
Lfs is better fit with large files not small icon png files IMO.

Can you consider to submit a PR ?

**Comment by kwizart** [comment]
Any issue ?

**Comment by thestonewell** [comment]
Yes time... I won't get to it before next week

**Comment by kwizart** [comment]
Fixed in 5.0.1 now in repos
Thx for the report.
